rotkehlchen/chain/ethereum/utils.py

```diff
@@ -166,3 +174,38 @@ def should_update_protocol_cache(cache_key: GeneralCacheType, *args: str) -> boo
             key_parts=[cache_key, *args],
         )
     return ts_now() - last_update_ts >= ETH_PROTOCOLS_CACHE_REFRESH
+
+
+def try_download_ens_avatar(
+        eth_inquirer: 'EthereumInquirer',
+        avatars_dir: Path,
+        ens_name: str,
+) -> None:
+    """
+    Handles ens avatar downloading.
+    1. Checks whether given ens name has avatar set
+    2. If it does -- downloads the avatar and saves it in `avatars_dir`
```

Comment:
bad habit to use `--` instead of a comma

```suggestion
    2. If it does, downloads the avatar and saves it in `avatars_dir`
```